src/algorithms.cpp

```diff
@@ -22,4 +24,105 @@ bool isComplete(const Model &model) {
     return true;
 }
 
+bool isLocCorrect(const Model& model, IdRef id){
+	const Node* node = model.node(id);
+
+	auto isInward = std::vector<bool>(Node::MAX_SLOTS, false);
+	auto isOutward = std::vector<bool>(Node::MAX_SLOTS, false);
```

Comment:
Use `std::array<bool, Node::MAX_SLOTS>` instead

Reply:
Changed in 97c800b4adaa6188f0b1479151af3b66fcfddb66